journalpump.spec

```diff
@@ -11,6 +11,10 @@ BuildRequires:  python3-devel, python3-pytest, python3-pylint python3-responses
 BuildArch:      noarch
 Obsoletes:      kafkajournalpump
 
+# The dependency generator uses names which are not compatible with Aiven
+# internal .spec files.
```

Comment:
I believe this is more related to Fedora package naming which I think is what we follow as well as much as possible. So maybe adjust that comment to refer to that.

Reply:
Thanks! Changed the sentence.